wp-content/themes/core/assets/pcss/theme.pcss

```diff
@@ -11,8 +11,11 @@
 
 /* Variables */
 @import "color/_variables.pcss";
+@import "typography/_variables.pcss";
 
 /* Mixins */
+@import "typography/_mixins.pcss";
 
 /* Utility Classes */
 @import "color/_utilities.pcss";
+@import "typography/_utilities.pcss";
```

Comment:
Since we're using the global-css plugin, I don't think we need to import the variables, mixins, or utilities like this. As long as they are added in the plugin's config, they should be available for use in any css entry point (including the theme, admin, & print entries).

Reply:
Without importing, the theme entrypoint doesn't have access to the variables. I know the path is right in `postcss.config.js` because it was giving me errors before.. I'll have to dig a bit deeper into the configuration. I wish that plugin had better docs.

Reply:
@dpellenwood I spent a half hour trying a couple different things with configuration, paths, and trying to disable preserving css properties with no luck.

I'm not at all familiar w/ how the build system works, how our `postcss.config.js` hooks in to WP scripts and how to modify the order of pcss plugins. If you've got time to explain it to me like I'm 5 or point me in the direction of docs you've found useful, I'd appreciate it!

In the mean time, mind if I merge this and we can put up a fix separately?